src/test/java/uk/gov/companieshouse/controller/AppealControllerTest_GET.java

```diff
@@ -56,16 +57,40 @@ public void whenAppealExists_return200() throws Exception {
     }
 
     @Test
-    public void whenAppealDoesNotExist_return404() throws Exception {
+    public void whenAppealDoesNotExistById_return404() throws Exception {
 
-        when(appealService.getAppeal(any(String.class))).thenReturn(Optional.empty());
+        when(appealService.getAppealById(any(String.class))).thenReturn(Optional.empty());
 
         mockMvc.perform(get(APPEALS_URI + "/{id}", TEST_COMPANY_ID, TEST_RESOURCE_ID)
             .contentType(MediaType.APPLICATION_JSON_VALUE))
             .andExpect(status().isNotFound())
             .andExpect(jsonPath("$").doesNotExist());
     }
 
+    @Test
+    public void whenAppealExistsByPenalty_return200() throws Exception {
+
+        when(appealService.getAppealByPenaltyReference(any(String.class), any(String.class))).thenReturn(Optional.of(getValidAppeal()));
```

Comment:
I'm not convinced here you've actually tested that the service query actually works because you've mocked the whole call. I would suggest writing a test to verify that the query does what it is intended